features/tracker-allowlist.json

```diff
@@ -468,6 +468,17 @@
                     }
                 ]
             },
+            "cmp.quantcast.com": {
```

Comment:
For context: platforms (at least the extension) treat the key as an eTLD+1 rather than hostname, so the extension wouldn't pick up on the mitigation if we used `"cmp.quantcast.com"` in the object's key